syntax = "proto3";

service Identity {
   rpc getIdentity (hrData) returns (iamData) {}
   rpc enterIdentity (hrData) returns (iamData) {}
}
message hrData {
   string hr_id = 1;
   string first_name = 2;
   string middle_name = 3;
   string last_name = 4;
   string job_title = 5;
   int32 location_number = 6;
   string location_name = 7;
   string manager_hr_id = 8;
}
message iamData {
   string hr_id = 1;
   string user_name = 2;
   string first_name = 3;
   string middle_name = 4;
   string last_name = 5;
   string job_title = 6;
   int32 location_number = 7;
   string location_name = 8;
   string manager_hr_id = 9;
}

// python -m grpc_tools.protoc --proto_path=. ./identity.proto --python_out=. --grpc_python_out=.